django/src/rdwatch_scoring/migrations/0001_initial.py

```diff
@@ -0,0 +1,1536 @@
+# Generated by Django 4.1.9 on 2023-06-22 08:08
+
+import django.contrib.gis.db.models.fields
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = []
```

Comment:
iirc it's best practice in Django to ensure your migrations always run in a deterministic order, so we can enforce that by doing
```suggestion
    dependencies = [
        ('rdwatch', '0010_hyperparameters_evaluation_and_more'),
    ]
```